utils/demo/helpers/addManipulationBindings.ts

```diff
@@ -55,15 +59,40 @@ export default function addManipulationBindings(toolGroup, register = true) {
         mouseButton: MouseBindings.Primary, // Shift Left Click
         modifierKey: KeyboardBindings.Shift,
       },
+      {
+        numTouchPoints: 1,
+        modifierKey: KeyboardBindings.Shift,
+      },
       {
         mouseButton: MouseBindings.Secondary,
       },
     ],
   });
+  // Need a binding to navigate without a wheel mouse
+  toolGroup.setToolActive(StackScrollTool.toolName, {
+    bindings: [
+      {
+        mouseButton: MouseBindings.Primary, // Shift Left Click
+        modifierKey: KeyboardBindings.Alt,
+      },
+      {
+        numTouchPoints: 1,
+        modifierKey: KeyboardBindings.Alt,
+      },
+    ],
+  });
+  // Add a length tool binding to allow testing annotations on examples targetting
+  // other use cases.  Use a primary button with shift+ctrl as that is relatively
+  // unlikely to be otherwise used.
   toolGroup.setToolActive(LengthTool.toolName, {
     bindings: [
       {
-        mouseButton: MouseBindings.Fifth_Button,
+        mouseButton: MouseBindings.Primary,
+        modifierKey: KeyboardBindings.ShiftCtrl,
+      },
+      {
+        numTouchPoints: 1,
+        modifierKey: KeyboardBindings.ShiftCtrl,
       },
     ],
   });
```

Comment:
We don't really need to add these by default. You can check out Leo's PR where he is including an option to disable them, which is a result of enabling them by default. If they are necessary, let's make them optional instead of the default setting.

Reply:
Length entirely, or StackScrollTool - or touch - which parts need to be optional.
I'm REALLY keen on having at least enough bindings to deal with the three basic options on a non mouse computer (eg touchpad).  Otherwise one can't effectively manage the examples needing scrolling or pan/zoom.  If that interferes with tool specific bindings, then we should figure out how to manage the tool specific ones as higher priority for overlapping/active buttons.